src/Components/Icon/Icon.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <v-icon class="custom-icon" :color="props.color">
+    <component :is="iconComponent" v-if="iconComponent" />
+    <span v-else>Icon not found</span>
```

Comment:
Do we need this? If we use an icon, we must pass an icon. Otherwise, no need for this component, right?